api/tests/test_cloud_record.py

```diff
@@ -170,6 +102,57 @@ def _saved_messages(self, message_path):
         """Return a list of messages under message_path."""
         return glob.glob(message_path)
 
+    def _make_record_post(self, message, expected_status,
+                          dn='/C=XX/O=XX/OU=XX/L=XX/CN=allowed_host.test',
+                          empaid='Test Process'):
+        """
+        Helper method to make a POST request.
+
+        By default,
+         - dn is /C=XX/O=XX/OU=XX/L=XX/CN=allowed_host.test
+         - empaid is 'Test Process'
+        """
+        with self.settings(QPATH=QPATH_TEST):
+
+            # Make the POST request
+            test_client = Client()
+            url = reverse('CloudRecordView')
+
+            if dn is not None:
+                # Include SSL_CLIENT_S_DN header
+                response = test_client.post(url,
+                                            message,
+                                            content_type="text/plain",
+                                            HTTP_EMPA_ID=empaid,
+                                            SSL_CLIENT_S_DN=dn)
+            else:
+                # Omit SSL_CLIENT_S_DN header
```

Comment:
added "Omit the SSL_CLIENT_S_DN header so the request is made without any certificate, and not authenticated"